apps/client/src/pages/CheckerPage.js

```diff
@@ -0,0 +1,231 @@
+   * @param { boolean } value - Set components to finished state or remove from finished state.
+   */
+  const setFinishedState = (component, value) => {
+    // If component is only a string, we make it a array first
+    const allComponents = Array.isArray(component) ? component : [component];
+
+    // If value is false, remove the components from the fishedComponents array
+    const newFinishedComponents =
+      typeof value === "boolean" && value === false
+        ? finishedComponents?.filter((c) => !allComponents.includes(c)) || []
+        : [...finishedComponents, ...allComponents];
+
+    // Save the new array to the context
+    sessionContext.setSessionData([
+      slug,
+      { finishedComponents: newFinishedComponents },
+    ]);
+  };
+
+  const isActive = (component, finished = false) => {
+    // If component is only a string, we make it a array first
+    const allComponents = Array.isArray(component) ? component : [component];
+
+    // If finished is true we check if it's finished, else check activeComponents.
+    const components = finished ? finishedComponents : activeComponents;
+    return components.includes(...allComponents) || false;
+  };
+
+  const isFinished = (component) => isActive(component, true);
+
+  /**
+   * Set the questionIndex the next questionId, previous questionId, or the given id.
+   *
+   * @param { int | ('next'|'prev') } value - This van be, 'next, prev or a int`
+   */
+  const goToQuestion = (value) => {
+    const newIndex = Number.isInteger(value)
+      ? value // Return the value if value isInteger()
+      : value === "next"
+      ? questionIndex + 1
+      : value === "prev"
+      ? questionIndex - 1
+      : console.error(
+          `goToQuestion(): ${value} is not an integer, 'next' or 'prev'`
+        );
+
+    sessionContext.setSessionData([
+      slug,
+      {
+        questionIndex: newIndex,
+      },
+    ]);
+  };
+
+  // Callback to go to the Conclusion section
+  // `false` is to prevent unexpected click, hover and focus states on already active section
+  const handleConclusionClick =
+    !isActive("conclusion") && isFinished("questions")
+      ? () => setActiveState("conclusion")
+      : false;
+
+  const checkedStyle = {
+    borderColor: "white",
+  };
+
+  return (
+    <Layout>
+      <Helmet>
+        <title>Vragen en conclusie - {text.heading}</title>
+      </Helmet>
+
+      <PrintDetails />
+
+      {/* STTR-flow with the StepByStepNavigation */}
+      {sttrFile && (
+        <StepByStepNavigation
+          customSize
+          disabledTextColor="inherit"
+          doneTextColor="inherit"
+          highlightActive
+          lineBetweenItems
+        >
+          <StepByStepItem
+            active={isActive("locationInput") || isActive("locationResult")}
+            checked={isActive("locationResult") || isFinished("locationResult")}
+            done={isActive("locationInput") || isActive("locationResult")}
```

Comment:
While reading this, I'm wondering... What does `done` actually do? I know active is the background color (and maybe related to scroll position) and `checked` is the chevron. `done` could be the blue line, but then it's value seems weird.